<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assessment_dialog</class>
 <widget class="QDialog" name="assessment_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Structure Assessment</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../../../../../icon.png</normaloff>../../../../../../../../../../../icon.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assessment Date</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_assessment_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assessment Description</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_assessment_date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>MM/dd/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2021</year>
     <month>12</month>
     <day>21</day>
    </date>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_structures">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Source</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_save_assessment">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel_assessment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_structure">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Structure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete_structure">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Structure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>148</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Structures</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
